docs/how-to-add-new-output.md

```diff
@@ -0,0 +1,67 @@
+# How to add a new output type
+
+## Understanding the API
+
+The ClusterLogForwarder API is defined by a collection of Go struct types.
+
+[type ClusterLogForwarderSpec struct][cluster_log_forwarder] specifies
+* inputs to select logs
+* outputs to forward logs to remote targets
```

Comment:
outputs to forward logs to remote targets using `OutputSpec` and `OutputTypeSpec`